applications/TrilinosApplication/CMakeLists.txt

```diff
@@ -28,6 +28,14 @@ endif()
 include_directories(SYSTEM ${TRILINOS_INCLUDE_DIR})
 link_directories(${TRILINOS_LIBRARIES})
 
+if(${TRILINOS_EXCLUDE_AMESOS2_SOLVER} MATCHES ON)
+    add_definitions(-DTRILINOS_EXCLUDE_AMESOS2_SOLVER)
+    message("**** WARNING: Manually disabled Amesos2-Solver ****")
+elseif(DEFINED ${HAVE_AMESOS2})
```

Comment:
@roigcarlo I cannot get this to work, tried several different things
=> should work in combination with `HAVE_AMESOS2` from `FindTRILINOS.cmake`

the idea is to automatically disable it if Amesos2 is not part of the trilinos installation

Can you have a look please?